packages/google_maps_flutter/lib/src/polygon.dart

```diff
@@ -67,6 +68,11 @@ class Polygon {
   /// default; to form a closed polygon, the start and end points must be the same.
   final List<LatLng> points;
 
+  /// The vertices of the holes to be cut out of polygon.
+  ///
+  /// Line segments of each points of hole are drawn inside polygon between consecutive hole points.
```

Comment:
Additional documentation as to what a hole is would be helpful. Also something about the sequence in which the lat lngs are supposed to be input. Something along the lines of:

To create an empty area within a polygon, you need to create two paths, one inside the other. To create the hole, the coordinates defining the inner path must be in the opposite order to those defining the outer path. For example, if the coordinates of the outer path are in clockwise order then the inner path must be counter-clockwise.

I copied it over from https://developers.google.com/maps/documentation/javascript/shapes#polygon_hole

Reply:
done